src/main/java/com/reshiftsecurity/plugins/intellij/actions/AnalyzeScopeFiles.java

```diff
@@ -271,10 +274,16 @@ public boolean visitFile(@NotNull final VirtualFile file) {
 	}
 
 	private static String getModuleNameInReadAction(@NotNull final Module module) {
-		return new ReadAction<String>() {
-			protected void run(@NotNull final Result<String> result) throws Throwable {
-				result.setResult(module.getName());
-			}
-		}.execute().getResultObject();
+//		return new ReadAction<String>() {
+//			protected void run(@NotNull Result<String> result) throws Throwable {
+//				result.setResult(module.getName());
+//			}
+//		}.execute().getResultObject();
```

Comment:
We could safely remove this block of code as opposed to leaving it commented out in git. We always have the git history to go back to the original code.